### PR TITLE
feat: add copy to clipboard buttons

### PR DESCRIPTION
**Pull request by Panzer1119**
This is my attempt to implement #541.

And here are some Screenshots from the various tabs:

### Error

Screenshot 2024-01-09 220457

### Data

Screenshot 2024-01-09 220514

### Options

Screenshot 2024-01-09 220527

### Logs

Screenshot 2024-01-09 220420

The buttons to copy a single line become visible only when hovering over a line.

### CODE REVIEW

**Comment by felixmosh** [comment]
Hi @Panzer1119, thank you for your contribution, I'll review the PR asap

File: packages/ui/src/components/JobCard/Details/DetailsContent/JobLogs/JobLogs.tsx [hunk 1]
```diff
@@ -127,6 +142,13 @@ export const JobLogs = ({ actions, job }: JobLogsProps) => {
                 data-line-number={`${log.lineNumber}.`}
               >
                 {log.message}
+                <Button
+                  onClick={copyLogToClipboard(log)}
```
Comment (felixmosh) [inline packages/ui/src/components/JobCard/Details/DetailsContent/JobLogs/JobLogs.tsx:146]: you've missed a function here, right?

**Review by felixmosh — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/JobCard/Details/DetailsContent/JobLogs/JobLogs.tsx (hunk 1 above)
Comment (Panzer1119) [reply packages/ui/src/components/JobCard/Details/DetailsContent/JobLogs/JobLogs.tsx:146]: Oh, yeah seems so.

Strange that it still worked for me.

Comment (felixmosh) [reply packages/ui/src/components/JobCard/Details/DetailsContent/JobLogs/JobLogs.tsx:146]: because it saved to clipboard on each render

**Comment by felixmosh** [comment]
closes #541

**Comment by felixmosh** [comment]
Thank you for you contribution. 
This feature released in `v5.12.0` 🎊

**Pull request merged**